recipes/mesa-glu/all/conanfile.py

```diff
@@ -0,0 +1,103 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import is_apple_os
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, load, rm, rmdir, save
+from conan.tools.gnu import PkgConfigDeps
+from conan.tools.layout import basic_layout
+from conan.tools.meson import Meson, MesonToolchain
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class MesaGluConan(ConanFile):
+    name = "mesa-glu"
+    description = "Mesa's implementation of the OpenGL utility library"
+    license = ("SGI-B-1.1", "SGI-B-2.0", "MIT")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.mesa3d.org/"
+    topics = ("gl", "glu", "mesa", "opengl")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    provides = "libGLU"
```

Comment:
```suggestion
```
We dont have such package, neither an open PR. Still, we will lowecase always.

Reply:
But `glu/system` provides the same library I'm pretty sure, at least on Linux.

Reply:
Then is should be `glu` instead:

https://docs.conan.io/2/reference/conanfile/attributes.html#provides

Reply:
Got it. I've committed your suggestion.